src/WebView.ios.tsx

```diff
@@ -381,6 +381,7 @@ class WebView extends React.Component<IOSWebViewProps, State> {
         scalesPageToFit={scalesPageToFit}
         // TODO: find a better way to type this.
         source={resolveAssetSource(this.props.source as ImageSourcePropType)}
+        contentInsetAdjustmentBehavior={this.props.contentInsetAdjustmentBehavior}
```

Comment:
not needed as it should be inside "otherProps"

Reply:
Done